storybook/src/elements/nuxeo-tree/nuxeo-tree.stories.js

```diff
@@ -0,0 +1,49 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-tree/nuxeo-tree.js';
+
+const data = {
+  title: 'Home',
+  children: [
+    {
+      title: 'Kitchen',
+      children: [],
+    },
+    {
+      title: 'Bedroom',
+      children: [
+        { title: 'Bed Frames', children: [] },
+        { title: 'Mattress', children: [] },
+      ],
+    },
+  ],
+};
+
+const controller = {
+  // How to get children of a node. Returns a promise.
+  getChildren(node) {
+    return Promise.resolve(node.children);
+  },
+  isLeaf(node) {
+    return node.children.length === 0;
+  },
+};
+
+storiesOf('UI/nuxeo-tree', module).add(
+  'Default',
+  () => html`
+    <style>
+      * {
+        font-family: 'Open Sans', Arial, sans-serif;
```

Comment:
Don't think we need this anymore after [ELEMENTS-1029](https://jira.nuxeo.com/browse/ELEMENTS-1029). (see https://github.com/nuxeo/nuxeo-elements/blob/master/storybook/.storybook/nuxeo-demo-theme.js#L241)

Reply:
done